test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -219,17 +219,33 @@ export default class BaseFormObject {
     }).exists;
   }
 
+  getAllAlertBox() {
+    return within(this.el).queryAllByRole('alert');
+  }
+
+  getAlertBox(index = 0) {
```

Comment:
nit:
```suggestion
  getNthAlertBox(index = 0) {
```
or `getAlertBoxByIndex`